src/App.js

```diff
@@ -33,6 +34,9 @@ function App() {
             <Route path="/add">
               <AddConference />
             </Route>
+            <Route path="/addURL">
```

Comment:
whats the reason behind not putting it in the add conference component?

Reply:
Hmmm, I guess they could go on the same page. Theres no reason, I just hadn't considered it.

Reply:
Okay just wondering if there was a reason for it! Makes the urls more readable if they're in the same place :)

Reply:
I'll approve you can decide how you want the structure to be :)